Neos.Neos/Resources/Private/Translations/en/Modules.xlf

```diff
@@ -489,6 +489,21 @@
 			<trans-unit id="packages.type" xml:space="preserve">
 				<source>Package Type</source>
 			</trans-unit>
+      <trans-unit id="packages.version" xml:space="preserve">
+				<source>Package Version</source>
```

Comment:
This is too long and breaks the header on smaller screens. We could make the version column larger, but IMO it's easier to shorten the header

```suggestion
				<source>Version</source>
```